### PR TITLE
RHDEVDOCS-3260 Changed triggers API version to v1beta1

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.9`, `enterprise-4.10`
- **JIRA issues**: [RHDEVDOCS-3260 Update docs for Triggers going Beta](https://issues.redhat.com/browse/RHDEVDOCS-3260)
- **Preview pages**: 
  - [Understanding OpenShift Pipelines > Triggers](https://deploy-preview-37744--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/understanding-openshift-pipelines.html#about-triggers_understanding-openshift-pipelines)
  - [Creating CI/CD solutions... > Adding triggers](https://deploy-preview-37744--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/creating-applications-with-cicd-pipelines.html#adding-triggers_creating-applications-with-cicd-pipelines)
- **SME review**: @savitaashture 
- **Peer-review**: @bburt-rh

### CODE REVIEW

**Review by savitaashture — COMMENTED** [review]
/lgtm

File: modules/op-about-pipelinerun.adoc [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+A _pipeline run_ is the running instance of a pipeline. It instantiates a pipeline for execution with specific inputs, outputs, and execution parameters on a cluster. A corresponding task run is created for each task automatically in the pipeline run.
 
-All the Tasks in the Pipeline are executed in the defined sequence until all Tasks are successful or a Task fails. The `status` field tracks and stores the progress of each TaskRun in the PipelineRun for monitoring and auditing purpose.
+All the tasks in the pipeline are executed in the defined sequence until all tasks are successful or a task fails. The `status` field tracks and stores the progress of each task run in the pipeline run for monitoring and auditing purpose.
 
-The following example shows a PipelineRun to run the `build-and-deploy` Pipeline  with relevant resources and parameters:
+The following example shows a pipeline run to execute the `build-and-deploy` pipeline  with relevant resources and parameters:
```
Comment (bburt-rh) [inline modules/op-about-pipelinerun.adoc:12]:
```suggestion
The following example runs the `build-and-deploy` pipeline with relevant resources and parameters:
```
- Per the IBM Style Guide, we should avoid using `execute`. 'Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.' My suggested edit above shortens the sentence by removing implied meaning and avoids using `execute`. 
- Also, extra space deleted

**Comment by bburt-rh** [comment]
One small suggestion, but otherwise lgtm

**Pull request merged**